activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -988,7 +988,7 @@ def invert_where
     end
 
     def invert_where! # :nodoc:
-      self.where_clause = where_clause.invert
+      self.where_clause = where_clause.invert if where_clause.present?
```

Comment:
thank you for the review 🙇‍♀️ 
as far as I look through, fixing [WhereClause#invert](https://github.com/rails/rails/blob/3926b8e5c792f273c406474709b02d28d2669f8e/activerecord/lib/active_record/relation/where_clause.rb#L85-L93) makes sense, and there seems no adverse effect on it.

so I fixed `invert` rather than `invert_where`.